src/backend/utils/storage.js

```diff
@@ -47,6 +51,13 @@ module.exports = {
 
   getFeedsCount: () => redis.scard(feedsKey),
 
+  setInvalidFeed: (id, reason) => {
+    const key = createInvalidFeedKey(id);
```

Comment:
File a follow-up issue to have us check to make sure the `id` exists in our database already, and throw if not.

Reply:
#654